source/core_pkcs11.c

```diff
@@ -134,6 +134,7 @@ CK_RV xGetSlotList( CK_SLOT_ID ** ppxSlotId,
     if( ( xResult != CKR_OK ) && ( pxSlotId != NULL ) )
     {
         PKCS11_FREE( pxSlotId );
+        *ppxSlotId = NULL;
```

Comment:
Could we add an assertion in test that the slot pointer returned is NULL.

Reply:
I don't think it is possible to stub out functions from the same source file.